### DISCUSSION TITLE
Create an issue case the workflow fails

### DISCUSSION THREAD

**Discussion by rffontenelle** [community/community#25111] [Actions]
<p>I have a workflow set to try to build a documentation and I would like to have issues opened whenever that workflow fails for some reason. Any suggestion on how I set this up?</p>

**Comment by MikeHamilton-RW** [comment, marked as answer]
<p>There are several ways I think you could approach this, some uglier than others. Is your document creation all in one job? And how many different steps? This Action looks like it would assist you along with the if conditional.</p>
<p><a href="https://github.com/JasonEtco/create-an-issue" rel="nofollow noopener">https://github.com/JasonEtco/create-an-issue</a></p>
<p><a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepsif" rel="nofollow noopener">https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepsif</a></p>
<pre><code>steps:

- name: Create Documentation
run: DO STUFF TO CREATE DOCS
  - name: Create Issue
if: failure() # OR Check a bool variable you create and toggle as needed
 uses:JasonEtco/create-an-issue@masterenv:
GITHUB\_TOKEN:${{ secrets.GITHUB\_TOKEN }}
</code></pre>
